### ISSUE TITLE
Two easy questions

### ISSUE THREAD

**Issue by PeterMcGor** [taehoonlee/tensornets#41]
Hello;

I am planning to compare the performance of a few segmentation models and I found the repo when I was looking for the easiest way to implement transfer learning from classifications to segmentation models. The work you have done until now is outstanding and for sure is helping a lot of us. 

I am just about to try tensornets but before I would like to ask a couple of questions in order to try to ease my own tasks. 

-   To implement skip connections, to formulate alternative loss functions and in general the usual machinery for segmentation models, I can have access to any level/layer/weight with `get_middles()` and `get()_outputs` of classification networks, right?

-  Is easy to integrate tensornets with [estimators](https://www.tensorflow.org/guide/estimators)? 

Thank you for you work!!

**Comment by taehoonlee** [comment]
Thank you for trying TensorNets, @PeterMcGor.
1. Yes, you are right.
2. I have never tried it. But, it looks easy :)
